tests/test_flows.py

```diff
@@ -568,30 +578,23 @@ def test_channel_selection(self):
             )
             time.sleep(1)
 
-            debug(
-                "Confirming channel selection request has been received on agent,step {}".format(i))
-
-            self.check_log(env.agents[0].radios[0], "CHANNEL_SELECTION_REQUEST_MESSAGE")
-            self.check_log(env.agents[0].radios[1], "CHANNEL_SELECTION_REQUEST_MESSAGE")
-
-            debug(
-                "Confirming tlvTransmitPowerLimit has been received with correct value on \
-                    agent, step {}".format(i))
+            debug("Confirming tlvTransmitPowerLimit has been received with correct value on agent,"
+                  " step {}".format(i))
 
             self.check_log(env.agents[0].radios[0], "tlvTransmitPowerLimit {}".format(tp20dBm))
-
-            self.check_log(env.agents[0].radios[0],
-                           "ACTION_APMANAGER_HOSTAP_CHANNEL_SWITCH_ACS_START")
-
             self.check_log(env.agents[0].radios[1], "tlvTransmitPowerLimit {}".format(tp20dBm))
 
-            self.check_log(env.agents[0].radios[0],
-                           "ACTION_APMANAGER_HOSTAP_CHANNEL_SWITCH_ACS_START")
-
             # TODO should be a single response (currently two are sent)
             self.check_cmdu_type("channel selection response", 0x8007, env.agents[0].mac,
                                  env.controller.mac, cs_req_mid)
 
+            (cur_chan_channel_0, _, _) = env.agents[0].radios[0].get_current_channel()
+            (cur_chan_channel_1, _, _) = env.agents[0].radios[1].get_current_channel()
+            if cur_chan_channel_0 != 6:
```

Comment:
This is probably a nitpick, but it is not clear to me where this `6` came from.
there is only a [comment](https://github.com/prplfoundation/prplMesh/pull/1360/commits/d32039cacc2b61f12ee2670587d77df6c6f44ee7#diff-f19fb9f58c705ea594e8e5c22bea2d5aR510) referencing it.
I don't know if using some variable would help here, because by just looking at the code I can't tell where this `6` can be replaced with a variable in the original `payload_wlan0`.

Reply:
The problem is that converting a channel number to the payload is quite complicated, not something you'd want in test code...

Maybe an additional comment would help here.